packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -172,6 +178,7 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
                 turns: _iconTurns,
                 child: const Icon(Icons.expand_more),
               ),
+              contentPadding: widget.padding,
```

Comment:
We tend to put `child` as the trailing named parameter. Put this new parameter above `child`.